demos/common/cpp/pipelines/src/async_pipeline.cpp

```diff
@@ -87,28 +89,27 @@ int64_t AsyncPipeline::submitData(const InputData& inputData, const std::shared_
                     {
                         auto blobPtr = request->GetBlob(outName);
 
-                        if(Precision::I32 == blobPtr->getTensorDesc().getPrecision())
+                        if (Precision::I32 == blobPtr->getTensorDesc().getPrecision())
                             result.outputsData.emplace(outName, std::make_shared<TBlob<int>>(*as<TBlob<int>>(blobPtr)));
                         else
                             result.outputsData.emplace(outName, std::make_shared<TBlob<float>>(*as<TBlob<float>>(blobPtr)));
                     }
 
                     completedInferenceResults.emplace(frameID, result);
-                    this->requestsPool->setRequestIdle(request);
+                    requestsPool->setRequestIdle(request);
                 }
                 catch (...) {
-                    if (!this->callbackException) {
-                        this->callbackException = std::current_exception();
+                    if (!callbackException) {
+                        callbackException = std::current_exception();
                     }
                 }
             }
-
             condVar.notify_one();
-    });
+        });
```

Comment:
```suggestion
    });
```

Reply:
it should be aligned to the left position of request->setCompletionCallback.
You may shift whole lambda body to the left, or just move closing bracket alignment.